src/plugins/intel_cpu/src/nodes/kernels/jit_kernel_traits.hpp

```diff
@@ -0,0 +1,32 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <dnnl_types.h>
+#include <dnnl_extension_utils.h>
+
+namespace ov {
+namespace intel_cpu {
+
+using namespace dnnl::impl;
+using namespace dnnl::impl::utils;
+namespace x64 = dnnl::impl::cpu::x64;
+
+template<x64::cpu_isa_t isa, ov::element::Type_t element_type>
+class jit_kernel_traits {
+public:
+    using Vmm = typename conditional3<isa == x64::sse41,
```

Comment:
Please add case for AVX as well

Reply:
Regarding `using Vmm` definition, there are about 90 occurrences of it in our code base. My understanding is that is more convenient to just use something like `using Vmm = typename x64::cpu_isa_traits<isa>::Vmm` instead.

`SIMD_WIDTH` is  commonly used name/definition in our code base and I believe it's hard to find kernel which doesn't use something like it. However, in this header this name depends on ov::element::Type_t, which makes this definition too specific.

So, I decided to exclude this file from PR. 
VCMPPS_LE and VCMPPS_GT constants were moved to jit_base. More constants like these will be added as needed.